.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+Short description explaining the high-level reason for the new issue.
+
+## Current behavior
+
+-
+
+## Expected behavior
+
+-
```

Comment:
Maybe not include the list item characters, here and above in current behavior? I doubt that this content will be a list, most of the time.